README.adoc

```diff
@@ -1,5 +1,175 @@
-judo-generator-commons
-----------------------
+# Judo Generator Commons
 
 JUDO Generator Commons project contains helpers, annotations and maven utils which can be used on a generator engine development
 
+== Description
+
+This is a general representation of the generator, which will be parameterized using one of the meta-models.
+We use the generator in this meta-models :
+
+- judo-meta-esm
+- judo-meta-pam
+- judo-meta-ui
+
+There are 3 sub-projects in these projects, which are related to the generator commons:
+
+- generator-engine
+- generator-maven-project
+- generator-maven-plugin-test
+
+The `generator-engine` instructs the generator commons how to release the items. This bind the meta-model to the generator.
+Here converted the model into template parameters.
```

Comment:
> Here converted the model into template parameters.

Models are converted/transformed into template parameter**s** in this project.

or

Models are converted/transformed into **a** template parameter in this project.